packages/simple-markdown/src/index.js

```diff
@@ -1789,7 +1789,7 @@ var htmlFor = function (outputFunc: HtmlNodeOutput): HtmlOutput {
 var outputFor = function (
     rules: OutputRules<Rule>,
     property: $Keys<Rule>,
-    defaultStat: ?State,
+    defaultState: ?State,
```

Comment:
I'd almost rather we put the default value up here if we can.  Otherwise it's buried down in the implementation online 1829.
```suggestion
    defaultState: ?State = {},
```